pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -591,40 +596,62 @@ func (s *Server) createIstioRA(opts *caOptions) (ra.RegistrationAuthority, error
 }
 
 // checkCABundleCompleteness checks if all required CA certificate files exist
-func checkCABundleCompleteness(signingKeyFile, signingCertFile, rootCertFile string, chainFiles []string) (signingCABundleComplete bool, bundleExists bool) {
-	signingKeyExists := fileExists(signingKeyFile)
-	signingCertExists := fileExists(signingCertFile)
-	rootCertExists := fileExists(rootCertFile)
-	chainFilesExist := hasValidChainFiles(chainFiles)
+func checkCABundleCompleteness(signingKeyFile, signingCertFile, rootCertFile string, chainFiles []string) (bool, bool, error) {
```

Comment:
more of a personal preference - I wonder if named returns here would be useful, since there are two bools that are pretty related. Will help make this more usable/extensible in the future